client/src/components/ElementEditor/tests/Summary-test.js

```diff
@@ -1,61 +1,45 @@
 /* eslint-disable import/no-extraneous-dependencies */
-/* global jest, describe, it, expect */
+/* global jest, test, describe, it, expect */
 
 import React from 'react';
 import Summary from '../Summary';
-import Enzyme, { shallow } from 'enzyme';
-import Adapter from 'enzyme-adapter-react-16';
-
-Enzyme.configure({ adapter: new Adapter() });
-
-describe('Summary', () => {
-  describe('render()', () => {
-    it('should render an image if the fileUrl prop is provided', () => {
-      const wrapper = shallow(
-        <Summary
-          fileUrl="/ss4/assets/Uploads/c70617f2e4/sample__FillWzEwMCwxMDBd.jpeg"
-          fileTitle="Sample Image"
-          content="Sample content"
-        />
-      );
-
-      expect(wrapper.find('img').length).toBe(1);
-    });
-
-    it('should not render an image if the fileUrl prop is not provided', () => {
-      const wrapper = shallow(
-        <Summary
-          fileUrl=""
-          fileTitle=""
-          content="Sample content"
-        />
-      );
-
-      expect(wrapper.find('img').length).toBe(0);
-    });
-  });
-
-  it('should render a content summary if the content is provided', () => {
-    const wrapper = shallow(
-      <Summary
-        fileUrl="/ss4/assets/Uploads/c70617f2e4/sample__FillWzEwMCwxMDBd.jpeg"
-        fileTitle="Sample Image"
-        content="Sample content"
-      />
-    );
+import { render } from '@testing-library/react';
+
+function makeProps(obj = {}) {
+  return {
+    fileUrl: '/ss4/assets/Uploads/c70617f2e4/sample__FillWzEwMCwxMDBd.jpeg',
+    fileTitle: 'Sample Image',
+    content: 'Sample content',
+    ...obj
+  };
+}
+
+test('Summary should render an image if the fileUrl prop is provided', () => {
+  const { container } = render(<Summary {...makeProps()}/>);
+  expect(container.querySelector('.element-editor-summary__thumbnail-image')).not.toBeNull();
```

Comment:
I think it's important here to check for an actual `img` element here, since we want to check an image is actually rendered in the DOM. Bonus points for validating the correct `src` even though the original didn't do that.

Reply:
Presumably thumbnail-image is an img

Reply:
We can be explicit instead of presuming - this is meant to be a test of what the user experiences, isn't it? It feels important to validate that the user is experiencing an image, and not just some random DOM element.

Reply:
Not doing

Reply:
After looking into this, it turns out the `target` and `container` props eventually get passed down to a reactstrap component, so it's okay to not test them here.